report-jtreg-comparator/src/main/java/io/jenkins/plugins/report/jtreg/main/comparator/Options.java

```diff
@@ -137,8 +142,33 @@ public void setPrintVirtual(boolean printVirtual) {
         this.printVirtual = printVirtual;
     }
 
+    public Configuration getConfiguration(String whatToFind) {
+        return configurations.get(whatToFind);
+    }
+
+    public void addConfiguration(String whatToFind, Configuration configuration) {
+        this.configurations.put(whatToFind, configuration);
+    }
+
     // enum of all available operations
     public enum Operations {
         List, Enumerate, Compare, Print
     }
+
+    public static class Configuration {
+        private final String configFileName;
```

Comment:
Maybe it should be file?

Reply:
I dont think so, because every build has different config file and that would mean saving every config file for every build. Now the tool just looks up the file in every build directory by the name.

Reply:
Sure, my wrong.  TY!